modyn/selector/selector.py

```diff
@@ -89,23 +89,35 @@ def inform_data_and_trigger(self, keys: list[int], timestamps: list[int], labels
 
         return trigger_id
 
-    def get_number_of_samples(self, trigger_id: int) -> int:
+    def get_number_of_samples(self, trigger_id: int) -> Tuple[int, float]:
         if trigger_id not in self._trigger_size_cache:
             raise ValueError(f"Trigger ID {trigger_id} does not exist!")
 
-        return self._trigger_size_cache[trigger_id]
+        # if we downsample the data, the dataset size is smaller
+        downsampling_scale = (
+            self._strategy.get_downsampling_scale() if isinstance(self._strategy, AbstractDownsampleStrategy) else 1.0
+        )
```

Comment:
I changed the function and put a comment explaining what happens

`
    def get_status_bar_scale(self) -> int:
        # the status bar scale is only meaningful if we are using a Downsampling strategy. Otherwise, it's always 100%
        if not isinstance(self._strategy, AbstractDownsampleStrategy):
            return 100

        return self._strategy.get_training_status_bar_scale()
`